langchain/src/agents/chat_convo/index.ts

```diff
@@ -127,11 +132,10 @@ export class ChatConversationalAgent extends Agent {
    * @param args.prefix - String to put before the list of tools.
    */
   static createPrompt(tools: Tool[], args?: CreatePromptArgs) {
-    const {
-      systemMessage = PREFIX,
-      humanMessage = SUFFIX,
-      outputParser = new AgentOutputParser(),
-    } = args ?? {};
+    const systemMessage =
```

Comment:
why is the prefix/suffix helpful here? its just same as system/human right? seems confusing to have two with same name

Reply:
The comment block above that line may be wrong (maybe copy-pasted from elsewhere?) but they implied that `args` should contain:
```js
   * @param args.suffix - String to put after the list of tools.
   * @param args.prefix - String to put before the list of tools.
```
I'm a bit new to Langchain's/prompt terminology in general so `prefix/suffix` seemed clearer but I can update the comment block and change the parameter names to `systemMessage` and `humanMessage` instead if that's preferred.

Reply:
I feel like we should deprecate these `systemMessage` and `humanMessage` properties to be in line with the other agents, or deprecate `prefix` on `suffix` on the other agents to avoid confusion here.